src/test/java/org/mvel2/tests/BaseMvelTest.java

```diff
@@ -0,0 +1,24 @@
+package org.mvel2.tests;
+
+import org.junit.Before;
+import org.mvel2.MVEL;
+import org.mvel2.optimizers.OptimizerFactory;
+
+/**
+ * Base class for Junit 4 TestCase.
+ */
+public class BaseMvelTest {
+
+    @Before
+    public void setUp() {
+        cleanUpConfigurations();
+    }
+
+    public static void cleanUpConfigurations() {
+        OptimizerFactory.setDefaultOptimizer(OptimizerFactory.DYNAMIC);
+        MVEL.COMPILER_OPT_ALLOW_NAKED_METH_CALL = false;
+        MVEL.COMPILER_OPT_ALLOW_OVERRIDE_ALL_PROPHANDLING = false;
+        MVEL.COMPILER_OPT_ALLOW_RESOLVE_INNERCLASSES_WITH_DOTNOTATION = false;
+        MVEL.COMPILER_OPT_SUPPORT_JAVA_STYLE_CLASS_LITERALS = false;
+    }
```

Comment:
I assume that this is the basic configuration. We can start with the clean configurations even if there are some test cases that change the configurations.